apis/tempo/v1alpha1/tempomonolithic_types.go

```diff
@@ -237,6 +237,13 @@ type MonolithicJaegerUISpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Route",order=4
 	Route *MonolithicJaegerUIRouteSpec `json:"route,omitempty"`
+
+	// Authentication defines the options for the oauth proxy used to protect jaeger UI
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Jaeger UI authentication configuration"
+	Authentication JaegerQueryAuthenticationSpec `json:"authentication,omitempty"`
```

Comment:
```suggestion
	Authentication *JaegerQueryAuthenticationSpec `json:"authentication,omitempty"`
```

In the monolithic CR all optional structs are pointers, otherwise they'll get saved/rendered as
```
authentication: {}
```
in the CR.